### PR TITLE
Fix "fixedPoint" formatting with specific precision (T848392)

### PR DESCRIPTION
**Pull request by DokaRus**
I've add more specific asserts to the shared localization tests.

### CODE REVIEW

File: testing/tests/DevExpress.localization/localization.utils.tests.js [hunk 1]
```diff
@@ -11,13 +11,15 @@ testModule('Localization utils', () => {
         assert.strictEqual(toFixed(4.64, 2), '4.64');
         assert.strictEqual(toFixed(35.855, 2), '35.86');
         assert.strictEqual(toFixed(35.855, 5), '35.85500');
-        assert.strictEqual(toFixed(-4.645, 2), '-4.65');
+        assert.strictEqual(toFixed(-4.645, 2), '-4.64');
```
Comment (alexander-kotov-dx) [inline testing/tests/DevExpress.localization/localization.utils.tests.js]: I think it is the wrong result. '-4.65' is correct

Comment (DokaRus) [reply testing/tests/DevExpress.localization/localization.utils.tests.js]: Chrome:
```
>new Intl.NumberFormat('en-US', { minimumFractionDigits: 2, maximumFractionDigits: 2 }).format(-4.645)
<"-4.65"
```
But:
```
>Globalize.locale( "en" );
>Globalize.numberFormatter({ minimumFractionDigits: 2, maximumFractionDigits: 2 })( -4.645 );
<"-4.64"
```
And IE:
```
>new Intl.NumberFormat('en-US', { minimumFractionDigits: 2, maximumFractionDigits: 2 }).format(-4.645)
<"-4.64"
```

Comment (DokaRus) [reply testing/tests/DevExpress.localization/localization.utils.tests.js]: Globalizejs still have a [problem](https://github.com/globalizejs/globalize/issues/884) with inaccurate floating point arithmetic. So you're right '-4.65' is correct

Comment (DokaRus) [reply testing/tests/DevExpress.localization/localization.utils.tests.js]: Looks like Globalizejs has a [problem](https://github.com/globalizejs/globalize/issues/884) with number rounding, so @alexander-kotov-dx  is correct

**Review by alexander-kotov-dx — DISMISSED** [review]

**Review by ksercs — DISMISSED** [review]

**Review by nightskylark — DISMISSED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**